src/DonorDashboards/resources/js/app/components/subscription-manager/index.tsx

```diff
@@ -95,14 +99,16 @@ const SubscriptionManager = ({id, subscription}) => {
 
     return (
         <div className={'give-donor-dashboard__subscription-manager'}>
-            <AmountControl
-                currency={subscription.payment.currency}
-                options={options}
-                max={max}
-                min={min}
-                value={amount}
-                onChange={setAmount}
-            />
+            {showAmountControls && (
+                <AmountControl
+                    currency={subscription.payment.currency}
+                    options={options}
+                    max={max}
+                    min={min}
+                    value={amount}
+                    onChange={setAmount}
+                />
+            )}
             <PaymentMethodControl
```

Comment:
I was thinking we should add a conditional for `gateway.can_update_payment_method` here as well, what do you think?

Reply:
Added a `showPaymentMethodControl` constant, defaulting it to the same value of `showAmountControl` in case the value of the prop is undefined (when not in the latest version of Recurring) keeping the same behavior users have now.

Reply:
thanks!